Anytype/Sources/PresentationLayer/TextEditor/Set/EditorSetViewModel.swift

```diff
@@ -177,10 +177,6 @@ final class EditorSetViewModel: ObservableObject {
         setDocument.setUpdatePublisher.sink { [weak self] in
             self?.onDataChange($0)
         }.store(in: &subscriptions)
-
-        Publishers.CombineLatest(setDocument.document.detailsPublisher, $isAppear)
-            .sink { [weak self] in self?.handleDetails(details: $0, isAppear: $1) }
-            .store(in: &subscriptions)
```

Comment:
do we need to maintain `isAppear` for any other reason?

Reply:
No. Deleted it